pkg/backend/aio_test.go

```diff
@@ -360,12 +360,12 @@ func TestBackEnd_ListAioControllers(t *testing.T) {
 		size    int32
 		token   string
 	}{
-		"valid request with invalid SPDK response": {
+		"valid request with empty result SPDK response": {
 			testAioVolumeID,
 			nil,
 			[]string{`{"id":%d,"error":{"code":0,"message":""},"result":[]}`},
-			codes.InvalidArgument,
-			fmt.Sprintf("Could not find any namespaces for NQN: %v", "nqn.2022-09.io.spdk:opi3"),
```

Comment:
how this passed before ?

Reply:
server returned OK status. And we checked against expectations only if returned error is not OK

Reply:
like here
https://github.com/opiproject/opi-spdk-bridge/blob/c2247af22c347f22af89845483342137e15d9c2c/pkg/backend/aio_test.go#L144

Reply:
understood